src/amo/components/CollectionManager/index.js

```diff
@@ -42,15 +42,15 @@ type Props = {|
 
 type InternalProps = {|
   ...Props,
-  clientApp: ?string,
+  clientApp: string,
```

Comment:
Do you know why these were marked as optional before? That suggests there is a case when they might be missing but now we're saying they will never be missing. Is that valid?